appHooks/apphub/prebuild.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Prebuild parse and test .apphub file
+ */
+import Promise from 'bluebird';
+import Hubkit from 'hubkit';
+import AppHook from '~/appHooks/appHook';
+
+const gh = new Hubkit();
+let hook = new AppHook('apphub', __dirname);
+
+hook.question(release => {
+  const application = release.ownerDocument();
+
+  return gh.request(`GET /repos/${application.github.fullName}/contents/.apphub?ref=${release.github.tag}`, {
+    token: application.github.APItoken,
+  })
+  .then(apphubData => {
+    try {
+      const apphubBase = new Buffer(apphubData.content, 'base64');
+      hook.meta(JSON.parse(apphubBase.toString()));
```

Comment:
it's just a function to store information for the issue. Mostly stuff like function dumps and stuff